apps/ecommerce/frontend/src/components/MultipleResources.tsx

```diff
@@ -38,62 +44,16 @@ const MultipleResources = () => {
     sdk.field.setValue(newValue);
   };
 
-  const mockValue: ExternalResourceLink = {
-    sys: {
-      urn: crypto.randomUUID(),
-      type: 'ResourceLink',
-      linkType: sdk.parameters.instance.linkType,
-    },
-    metadata: {
-      resourceType: 'Commerce:Product',
-    },
-  };
-
   return (
-    <Grid rowGap="spacingM">
-      {value && (
-        <SortableLinkList<ExternalResourceLink>
-          items={value}
-          axis="y"
-          useDragHandle={true}
-          isInitiallyDisabled={true}
-          isDisabled={false}
-          hasCardEditActions={false}
-          sdk={sdk}
-          viewType={'card'}
-          parameters={{ instance: {} }}>
-          {({ item, isDisabled, DragHandle, index }) => {
-            return (
-              <Grid.Item>
-                <ResourceCard
-                  key={index}
-                  index={index}
-                  total={total}
-                  value={item}
-                  dragHandleRender={DragHandle}
-                  onRemove={onRemove}
-                  onMoveToTop={onMoveToTop}
-                  onMoveToBottom={onMoveToBottom}
-                />
-              </Grid.Item>
-            );
-          }}
-        </SortableLinkList>
-      )}
-      <Grid.Item>
-        <AddContentButton
-          onClick={() => sdk.field.setValue(value ? [...value, mockValue] : [mockValue])}
-        />
-      </Grid.Item>
-      <Grid.Item>
-        <TextLink as="button" onClick={() => setIsExpanded(!isExpanded)}>
-          {isExpanded ? 'Hide' : 'Show'} JSON
-        </TextLink>
-        <Collapse isExpanded={isExpanded}>
-          <Field sdk={sdk} />
-        </Collapse>
-      </Grid.Item>
-    </Grid>
+    <ResourceList
+      addContent={addContent}
+      isMultiple={true}
+      onMoveToBottom={onMoveToBottom}
```

Comment:
Annoying nit: One convention I think we should follow is that upon creating the function the name should be 'handle'. When it's passed as a prop we name it with the `on` prefix

Reply:
All for this!